### ISSUE TITLE
2.3.0 accessibility: browser language preference not reflected on fresh conference

### ISSUE THREAD

**Issue by schrd** [bigbluebutton/bigbluebutton#12246]
**Describe the bug**

If you start a fresh conference, the lang attribute keeps the value en regardless of the browser preference. It is changed only if you change the language in the settings dialog. Therefore screen readers prefix all output with "Englisch" if your browser prefers german.

This is a follow up of #12092 

**To Reproduce**

Change your browser preferences to prefer any language != english. Start a conference. Open element inspector, inspect the `lang` attribute of the `<html>` tag. 

**Expected behavior**
The `lang` attribute should be changed if bbb-html5 changes the ui language base on browser preferences.

**BBB version (optional):**
2.3.0
